test/route.go

```diff
@@ -19,8 +19,11 @@ limitations under the License.
 package test
 
 import (
+	"net/http"
+	"strings"
 	"testing"
 
+	"github.com/knative/pkg/test/spoof"
```

Comment:
Should we move the new method into this package, e.g. in `requests.go`?

Reply:
Eventually maybe. It might be heavily debated if we want to do this at all though and request.go lives in knative/pkg so I opted to only have it in serving for now to not advertise it's usage too much.